test/unit/voldemort/server/VoldemortServerTest.java

```diff
@@ -74,16 +74,24 @@ private VoldemortServer getVoldemortServer(Properties properties) throws IOExcep
     }
 
     @Test
-    public void testJCEProvider() throws IOException {
+    public void testSSLProviders() throws IOException {
+        // The SSL providers changes the state of the JVM
+        // if the methods are run in parallel or the ordering
+        // changes the test fails. TestBouncyCastle is the
+        // only test to enable this SSL. If other tests introduce
+        // them the test could fails as well.
```

Comment:
This is something I regressed(It is in the right way in the master). The current one has a single method for it, I tried to improve, but regressed it. This time, I am adding a comment to prevent further regressions :) https://github.com/voldemort/voldemort/blob/master/test/unit/voldemort/server/VoldemortServerTest.java#L28